woocommerce/woocommerce-android
# Feature/9804 subscreen about your product

Closes: #9804


### Description

Adds UI for "About your product" step in the product creation with AI flow. 

### Testing instructions

1. Click on "Add Product" button from products list and select add product with AI. The site must be WP.com to be eligible for this option. 
2. Click "Continue" in the first step. 
3. Check the UI matches the expected design
